apps/storybook/src/DatePicker.stories.tsx

```diff
@@ -208,3 +208,64 @@ WithYear.args = {
   date: new Date(2021, 4, 11, 14, 55, 22),
   showYearSelection: true,
 };
+
+export const Range: Story<DatePickerProps> = (args) => {
+  const {
+    setFocus = true,
+    localizedNames,
+    startDate = new Date(2022, 6, 13, 14, 55, 22),
+    endDate = new Date(2022, 6, 27, 14, 55, 22),
+    ...rest
+  } = args;
+  const [opened, setOpened] = React.useState(false);
+  const [currentStartDate, setCurrentStartDate] = React.useState(
+    new Date(startDate),
```

Comment:
Do you need to create new Date here since Date comes from the props?

Reply:
Not sure. I saw the other DatePicker stories initialized the current date this way. I've removed it

Reply:
Do that for other parts in this story too.

Reply:
Changed